.github/workflows/format.yml

```diff
@@ -7,7 +7,11 @@ jobs:
         runs-on: ubuntu-latest
         steps:
             - uses: actions/checkout@v4
-            - uses: actionsx/prettier@v2
+            - uses: actions/setup-node@v3
               with:
-                  # We should consider autofixing the issues at some point
-                  args: --check .
+                  node-version: "18"
```

Comment:
We recently updated the node version https://github.com/Cognigy/chat-components/pull/107/files